packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/QuestionCard.tsx

```diff
@@ -49,151 +56,206 @@ const QuestionCard: React.FC<QuestionCardProps> = ({
+                      overflow: 'hidden',
+                      maxWidth: '55em',
+                    } as React.CSSProperties
+                  }
+                >
+                  {question.text}
+                </div>
+              </Tooltip>
+            )}
+            {isStaff && (
+              <div className="mr-1 mt-0.5 inline-block min-w-[120px] text-sm italic text-gray-600">
+                {question.creator.name}
               </div>
-            </Tooltip>
-          )}
-          {isStaff && (
-            <div className="mr-1 mt-0.5 inline-block min-w-[120px] text-sm italic text-gray-600">
-              {question.creator.name}
-            </div>
-          )}
+            )}
 
-          {question.questionTypes?.map((questionType, index) => (
-            <QuestionTagElement
-              key={index}
-              tagName={questionType.name}
-              tagColor={questionType.color}
-            />
-          ))}
-        </Col>
-        {isBeingHelped && !isStaff && question.helpedAt && (
-          <Col flex="0 0 3rem">
-            <div className="text-sm font-medium text-green-700">
-              {servedTime}
-            </div>
+            {question.questionTypes?.map((questionType, index) => (
+              <QuestionTagElement
+                key={index}
+                tagName={questionType.name}
+                tagColor={questionType.color}
+              />
+            ))}
           </Col>
-        )}
-        <Col flex="0 0 3rem">
-          <div className="text-sm text-gray-600">{getWaitTime(question)}</div>
-          {isStaff && isBeingHelped && (
-            <div className="text-sm font-medium text-green-700">
-              {servedTime}
-            </div>
-          )}
-        </Col>
-        {isStaff && (
-          <Col className="w-full sm:w-auto">
-            <TAQuestionCardButtons
-              courseId={cid}
-              queueId={qid}
-              question={question}
-              hasUnresolvedRephraseAlert={false}
-              tasksSelectedForMarking={tasksSelectedForMarking}
-              className="align-center flex items-center justify-around"
-            />
+          <Col flex={'0.1 1 auto'}>
+            {(isBeingHelped || isPaused) && !isStaff && (
+              <Row justify={'end'}>
+                <div
+                  className={cn(
+                    'text-sm',
+                    isPaused ? 'text-amber-400' : '',
+                    isBeingHelped ? 'text-green-700' : '',
+                  )}
+                >
+                  {isPaused && 'Currently Paused'}
+                  {isBeingHelped && 'Currently Being Served'}
+                </div>
+              </Row>
+            )}
+            <Row
+              justify={'end'}
+              className={cn(
+                !isBeingHelped && !isPaused ? 'h-[2.5rem]' : '',
+                'gap-1',
+              )}
+            >
+              <Col flex="1 0 3rem">
```

Comment:
3rem might be a little too much for flex-basis, since it kinda cuts off the text a little early here
![image](https://github.com/user-attachments/assets/c431b990-ed7b-4fca-9362-b4d38d817723)